docs/how-to/incremental-migrations.md

```diff
@@ -0,0 +1,365 @@
+# Incremental Migrations
+
+SpacetimeDB does not contain support for general schema-modifying migrations. It does, however, allow adding new tables, and changing reducers' definitions in arbitrary ways. It's possible to run general migrations using an external tool, but this is tedious, necessitates downtime, and imposes the requirement that you update all your clients at the same time as publishing your new module version.
```

Comment:
```suggestion
SpacetimeDB does not yet provide built-in support for general schema-modifying migrations. It does, however, allow adding new tables, and changing reducers' definitions in arbitrary ways. It's possible to run general migrations using an external tool, but this is tedious, necessitates downtime, and imposes the requirement that you update all your clients at the same time as publishing your new module version.
```

Reply:
I'm comfortable saying "provide built-in support" rather than "contain support," but I don't think we should say "yet" here. As I noted in the style guide, I think we need to keep our plans for future features confined to the road map, so that our docs can be concerned with the state of the world as it is today. When we use phrasings like "does not yet provide," it gives readers the sense that we're apologizing for known flaws in SpacetimeDB. We want to present SpacetimeDB 1.0 as a complete tool that's sufficient for building real-world applications already, and that means owning the choices we've made not to provide certain features.